src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -114,9 +114,11 @@ public function mkdir($dirs, $mode = 0777)
      */
     public function exists($files)
     {
+        $maxPathLength = '\\' === DIRECTORY_SEPARATOR ? 258 : PHP_MAXPATHLEN;
```

Comment:
instead of using this ternary op, I propose to unconditionnalt, always use `PHP_MAXPATHLEN - 2`

Reply:
@nicolas-grekas  This proposal is very good, I have made the appropriate changes.